keras/utils/conv_utils_test.py

```diff
@@ -77,6 +77,18 @@ def test_normalize_tuple(self):
     with self.assertRaises(ValueError):
       conv_utils.normalize_tuple(None, n=3, name='strides')
 
+  def test_normalize_positive_tuple(self):
+    self.assertEqual((1, 2, 3,),
+                     conv_utils.normalize_positive_tuple((1, 2, 3), n=3, name='pool_size'))
+    self.assertEqual((3, 3, 3),
+                     conv_utils.normalize_positive_tuple(3, n=3, name='pool_size'))
+
+    with self.assertRaises(ValueError):
```

Comment:
Please check the content of the error message as well.

Reply:
Done